src/handlers/snsSubCustomerCreated.js

```diff
@@ -0,0 +1,9 @@
+
+
+exports.handler= async (event,context)=>{
```

Comment:
Minor note but the formatting is off, i.e. = sign is straight after handler.
```
exports.handler = async (event, context) => {
```

Ideally should be formatted as above with the spaces. If you do CMD + SHIFT + F in Vscode it should sort it out :)